include/nbl/video/SPhysicalDeviceFeatures.h

```diff
@@ -33,15 +33,15 @@ struct SPhysicalDeviceFeatures
     /* Vulkan 1.0 Core  */
     bool robustBufferAccess = false;
     bool fullDrawIndexUint32 = false;
-    bool imageCubeArray = false;
+    bool imageCubeArray = false; //always enable
```

Comment:
I'm open to suggestions about what features we should move to limits (and always enable if present).

For example, a bad idea to always enable are:
- anything to do with robustness
- tesellation & geometry shader
- anything added as compat
- dynamic state

Good things to always enable:
- drawIndirectFirstInstance
- imageCubeArrays